server/routes/user.py

```diff
@@ -148,7 +148,7 @@ def register_user():
         )
 
         # Build link for frontend verify page
-        frontend_url = os.getenv("FLASK_CLIENT_URL", "http://localhost:5173")
+        frontend_url = os.getenv("VITE_FRONTEND_URL", "http://localhost:5177")
```

Comment:
Return this naming to its original 
Keep all variables in flask to start with the FLASK keyword